Plugins/GitUIPluginInterfaces/GitUIEventArgs.cs

```diff
@@ -9,12 +9,13 @@ public class GitUIEventArgs : CancelEventArgs
     {
         private readonly IFilteredGitRefsProvider _getRefs;
 
-        public GitUIEventArgs(IWin32Window? ownerForm, IGitUICommands gitUICommands, Lazy<IReadOnlyList<IGitRef>> getRefs)
+        public GitUIEventArgs(IWin32Window? ownerForm, IGitUICommands gitUICommands, Lazy<IReadOnlyList<IGitRef>> getRefs, bool isForceUpdate)
```

Comment:
```suggestion
        public GitUIEventArgs(IWin32Window? ownerForm, IGitUICommands gitUICommands, Lazy<IReadOnlyList<IGitRef>> getRefs, bool forceRefresh)
```

Reply:
will change this (and other comments)
plan to move this to a separate class as well, this is an odd fit here.

Reply:
done